ethtx_ce/app/api/decorators.py

```diff
@@ -31,6 +31,8 @@
 
 log = logging.getLogger(__name__)
 
+jsonpickle.set_decoder_options('simplejson', use_decimal=True)
```

Comment:
We should add simplejson to the dependencies. Ethtx_ce can be developed independently

Reply:
It is already included in the Pipfile [https://github.com/EthTx/ethtx_ce/blob/master/Pipfile](Pipfile), do you mean to explicitly import it in the current python file?